core/modules/system/system.theme.inc

```diff
@@ -393,37 +393,24 @@ function theme_status_report($variables) {
  * @param $variables
  *   An associative array containing:
  *   - blocks: An array of blocks to display. Each array should include a
- *     'title', a 'description', a formatted 'content' and a 'position' which
- *     will control which container it will be in. This is usually 'left' or
- *     'right'.
+ *     'title', a 'description' and a formatted 'content'.
  *
  * @ingroup themeable
  */
 function theme_admin_page($variables) {
   $blocks = $variables['blocks'];
-
-  $stripe = 0;
   $container = array();
 
   foreach ($blocks as $block) {
     if ($block_output = theme('admin_block', array('block' => $block))) {
-      if (empty($block['position'])) {
-        // perform automatic striping.
-        $block['position'] = ++$stripe % 2 ? 'left' : 'right';
-      }
-      if (!isset($container[$block['position']])) {
-        $container[$block['position']] = '';
-      }
-      $container[$block['position']] .= $block_output;
+      $container[] = $block_output;
     }
   }
 
   if (!empty($container)) {
     $output = '<div class="admin clearfix">';
-      foreach ($container as $id => $data) {
-        $output .= '<div class="' . $id . ' clearfix">';
+      foreach ($container as $data) {
```

Comment:
@BWPanda this coding standards violation needs to be fixed.

Reply:
...never mind. I was looking at the commit instead of the entire PR. Please ignore, and sorry 😬